tests/lit-tests/1210.ispc

```diff
@@ -0,0 +1,5 @@
+// RUN: %{ispc} %s -MMM %t.dep -h %t.h > %t 2>&1
+// RUN: FileCheck --input-file=%t %s --implicit-check-not "writing dependencies"
```

Comment:
Shouldn't we check for this string starting from capital letter not?

Reply:
Good catch. I've modified it.